perllib/FixMyStreet/App/Form/Widget/Field/CheckboxGroup.pm

```diff
@@ -0,0 +1,60 @@
+package FixMyStreet::App::Form::Widget::Field::CheckboxGroup;
+
+use Moose::Role;
+with 'HTML::FormHandler::Widget::Field::CheckboxGroup';
+use namespace::autoclean;
+
+sub render_element {
+    my ( $self, $result ) = @_;
+    $result ||= $self->result;
+
+    my $output = '<ul class="permissions-checkboxes">';
```

Comment:
If we're rendering this ourselves, any scope to include the all/none shortcuts?
![image](https://user-images.githubusercontent.com/4776/58244631-e7b59600-7d4a-11e9-97d7-b6ef5771e103.png)